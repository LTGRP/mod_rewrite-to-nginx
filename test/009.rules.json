[
    {
        "type": "rewrite",
        "match": "^(.*)\\.(ico|jpg|png|gif)$",
        "target": "files/$1.$2",
        "conditions": "!-f $request_filename"
    }
]
