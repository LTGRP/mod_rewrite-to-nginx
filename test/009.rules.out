if (!-f $request_filename) {
    rewrite ^(.*)\.(ico|jpg|png|gif)$ files/$1.$2 last;
}
